candle-core/src/metal_backend/mod.rs

```diff
@@ -1282,17 +1276,17 @@ impl BackendStorage for MetalStorage {
                 dst.dtype()
             )
         }
-        let command_buffer = self.device.command_buffer()?;
         if src_s == d2 && dst_s == d2 {
-            command_buffer.set_label("copy2d_contiguous");
-            let blit = command_buffer.new_blit_command_encoder();
-            blit.set_label("copy2d_contiguous");
+            self.device.end_compute_encoding()?;
```

Comment:
Same question as in other comments, would be great for all `end_compute_encoding` to be documented.